### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by bkhan2**
I made some changes to the README to help new starters, like me, spin up sdx.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,14 +11,19 @@ This repository allows you to spin up a local test environment for the ``sdx-``
...
 
 ### Getting started
+Make a projects directory e.g. `/Users/[NAME]/projects`
+In the ``projects`` make a directory named "sdx". this is where the SDX services will be.
 
-Export a ``SDX_HOME`` environment variable. This should point at a folder into
-which you wish to clone the rest of the services (e.g. ``/home/my-user/sdx``)
+Export a ``SDX_HOME`` environment variable.
 [see [configuration](#configuration)]
+HOW-TO:
```
Comment (insacuri) [inline README.md]: I wouldn't bother with the 'how to use vi' section.

File: README.md [hunk 2]
```diff
@@ -11,14 +11,19 @@ This repository allows you to spin up a local test environment for the ``sdx-``
  - python3
 
 ### Getting started
+Make a projects directory e.g. `/Users/[NAME]/projects`
+In the ``projects`` make a directory named "sdx". this is where the SDX services will be.
 
-Export a ``SDX_HOME`` environment variable. This should point at a folder into
-which you wish to clone the rest of the services (e.g. ``/home/my-user/sdx``)
+Export a ``SDX_HOME`` environment variable.
```
Comment (insacuri) [inline README.md]: I'd change this to:
There needs to be an SDX_HOME environment variable pointed to where you will have all the SDX repos checked out to.  To do this, add   `export SDX_HOME=$HOME/projects/sdx` (the location is an example but you can have it checked out anywhere you want) to your .bash_profile (if using bash).

File: README.md [hunk 3]
```diff
@@ -11,14 +11,19 @@ This repository allows you to spin up a local test environment for the ``sdx-``
...
 [see [configuration](#configuration)]
+HOW-TO:
+Run this command in terminal -> ``vi ~/.bash_profile``
+Press `i` to edit the text and paste this on a new line -> ``export SDX_HOME=$HOME/projects/sdx``
+Type ``:wq`` and press Enter. This writes the changes and exits the editor.
 
 Set up a virtual environment using Python3 and ensure you activate it.
 
-Clone this repository somewhere and run ``make``
+cd into `/Users/[NAME]/projects/sdx` and run the following:
```
Comment (insacuri) [inline README.md]: Change this to:
Do `cd $SDX_HOME` and run the following:

File: README.md [hunk 4]
```diff
@@ -11,14 +11,19 @@ This repository allows you to spin up a local test environment for the ``sdx-``
  - python3
 
 ### Getting started
+Make a projects directory e.g. `/Users/[NAME]/projects`
```
Comment (insacuri) [inline README.md]: Remove these 2 lines for the change I mentioned below

File: README.md [hunk 5]
```diff
@@ -11,14 +11,19 @@ This repository allows you to spin up a local test environment for the ``sdx-``
...
-Export a ``SDX_HOME`` environment variable. This should point at a folder into
-which you wish to clone the rest of the services (e.g. ``/home/my-user/sdx``)
+Export a ``SDX_HOME`` environment variable.
 [see [configuration](#configuration)]
+HOW-TO:
+Run this command in terminal -> ``vi ~/.bash_profile``
+Press `i` to edit the text and paste this on a new line -> ``export SDX_HOME=$HOME/projects/sdx``
+Type ``:wq`` and press Enter. This writes the changes and exits the editor.
 
 Set up a virtual environment using Python3 and ensure you activate it.
```
Comment (insacuri) [inline README.md]: Can you remove this line, it's not true.

**Review by insacuri — CHANGES_REQUESTED** [review]

**Comment by bkhan2** [comment]
Changes made

File: README.md [hunk 6]
```diff
@@ -11,21 +11,24 @@ This repository allows you to spin up a local test environment for the ``sdx-``
...
+Make a directory called `sdx` e.g. `/users/[your_name]/sdx`.
 
-Export a ``SDX_HOME`` environment variable. This should point at a folder into
-which you wish to clone the rest of the services (e.g. ``/home/my-user/sdx``)
+This will contain all the files for the sdx services,
+
+Export an environment variable called ``SDX_HOME`` to the `sdx` folder.
 [see [configuration](#configuration)]
 
+Open your .bash_profile and add the line: ``export SDX_HOME=$HOME/[LOCATION OF SDX FOLDER]``
```
Comment (insacuri) [inline README.md]: This line and the other changed line (line 18) needs to change.  The line 18 line doesn't make any sense.  This line isn't accurate either.  Assuming you follow the instructions you'd have SDX_HOME be `$HOME/users/name/sdx` which doesn't exist.

Just change these lines to:
Open your .bash_profile and add the line: `export SDX_HOME=[LOCATION OF SDX FOLDER]`
Then apply the changes to the bash_profile to your session by doing `. ~/.bash_profile`

**Review by insacuri — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -11,21 +11,23 @@ This repository allows you to spin up a local test environment for the ``sdx-``
...
+This will contain all the files for the sdx services,
 
-Export a ``SDX_HOME`` environment variable. This should point at a folder into
-which you wish to clone the rest of the services (e.g. ``/home/my-user/sdx``)
 [see [configuration](#configuration)]
 
-Set up a virtual environment using Python3 and ensure you activate it.
+Open your .bash_profile and add the line: ``export SDX_HOME=[LOCATION OF SDX FOLDER]``
+
+Apply these changes to your bash_profile by doing ``~/.bash_profile``
```
Comment (insacuri) [inline README.md]: This line is wrong, the command is:
`. ~/.bash_profile`

Also, why are you using pairs of backticks?  Use either 1 or 3

**Review by insacuri — CHANGES_REQUESTED** [review]

**Review by insacuri — APPROVED** [review]

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**